### PR TITLE
Olha/8 cicd developer branch

### PR DESCRIPTION
**Pull request by KhOlga**

### CODE REVIEW

File: kubernetes/namespace.yml [hunk 1]
```diff
@@ -4,14 +4,14 @@ kind: Namespace
...
+#apiVersion: v1
+#kind: Namespace
+#metadata:
+#  name: staging
+#
+#---
+#apiVersion: v1
+#kind: Namespace
+#metadata:
+#  name: production
```
Comment (AISlearning) [inline kubernetes/namespace.yml]:
```suggestion
---
apiVersion: v1
kind: Namespace
metadata:
  name: staging

---
apiVersion: v1
kind: Namespace
metadata:
  name: production
```

File: .github/workflows/k8s_deployment_development.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        role-to-assume: arn:aws:iam::851725332718:role/eks_github_oidc-team-1-fp-eks-9deZbI34
+        aws-region: eu-central-1
+        role-session-name: GithubActionsSession
+
+    - name: Setup connection with AWS EKS
+      run: aws eks update-kubeconfig --name $EKS_CLUSTER_NAME --region $AWS_REGION --role-arn $ROLE_ARN
+
+    - name: Switch namespace to testing
+      run: kubectl config set-context --current --namespace=development
+
```
Comment (AISlearning) [inline .github/workflows/k8s_deployment_development.yml:36]:
```suggestion

    - name: Alter configmap and apply
      run: |
        kubectl apply -f ./kubernetes/ingress
        sleep 30s
        export BACKEND_HOSTNAME=$(kubectl -n development get ingress ingress  -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
        envsubst < ./kubernetes/frontend/10_config_map.yml | kubectl apply -f -
        
```

**Review by AISlearning — CHANGES_REQUESTED** [review]
Looks good, can you apply the changes requested?

**Comment by KhOlga** [comment]
Changes applyed

**Review by AISlearning — APPROVED** [review]

**Pull request merged**